python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -125,9 +125,15 @@ def stop_ganga():
 
         from Ganga.GPI import jobs, templates
         for j in jobs:
-            j.remove()
+            try:
+                j.remove()
+            except:
+                pass
         for t in templates:
-            t.remove()
+            try:
+                t.remove()
+            except:
+                pass
```

Comment:
Are these exception handlers needed? I haven't seen problems myself. Otherwise they should use a more specific handler.